src/lib/integration_test_lib/node_id.ml

```diff
@@ -0,0 +1,18 @@
+(* HOLD UP: I think we don't need this if we introduce a `From_node.t` type outside. *)
+
+(* Node ids are represented as strings in all integration test engines.
```

Comment:
Don't see this used. we are using pod_id in Node.t  instead.

Reply:
Yes, sorry, let me delete this. I started down this path, and then realized I didn't need this.